docs/integrations/hdinsight.md

```diff
@@ -0,0 +1,182 @@
+
+In Hopsworks, click on your *username* in the top-right corner and select *Settings* to open the user settings. Select *API keys*. Give the key a name and select the project scope before creating the key. Make sure you have the key handy for the next steps.
+
+!!! success "Scopes"
+    The API key should contain at least the following scopes:
+
+    1. featurestore
+    2. project
+    3. job
+
+<p align="center">
+  <figure>
+    <img src="../../../assets/images/azure/hdinsight/step-0.png" alt="Generating an API key on Hopsworks">
+    <figcaption>API keys can be created in the User Settings on Hopsworks</figcaption>
+  </figure>
+</p>
+
+!!! info
+    You are only able to retrieve the API key once. If you forget to copy it to your clipboard, delete it and create a new one.
+
+## Step 2:  Use a script action to install the Feature Store connector
+
+HDInsight requires Hopsworks connectors to be able to communicate with the Hopsworks Feature Store. These connectors can be installed with the script action shown below. Copy the content into a file, name the file `hopsworks.sh` and replace MY_INSTANCE, MY_PROJECT, MY_VERSION, MY_API_KEY and MY_CONDA_ENV with your values. Copy the `hopsworks.sh` file into any storage that is readable by your HDInsight clusters and take note of the URI of that file e.g., `https://account.blob.core.windows.net/scripts/hopsworks.sh`.
+
+The script action needs to be applied head and worker nodes and can be applied during cluster creation or to an existing cluster. Ensure to persist the script action so that it is run on newly created nodes. For more information about how to use script actions, see [Customize Azure HDInsight clusters by using script actions](https://docs.microsoft.com/en-us/azure/hdinsight/hdinsight-hadoop-customize-cluster-linux).
+
+!!! attention "Matching Hopsworks version"
+    The **major version of `HSFS`** needs to match the **major version of Hopsworks**. Check [PyPI](https://pypi.org/project/hsfs/#history) for available releases.
+
+    <p align="center">
+        <figure>
+            <img src="../../assets/images/hopsworks-version.png" alt="HSFS version needs to match the major version of Hopsworks">
+            <figcaption>You find the Hopsworks version inside any of your Project's settings tab on Hopsworks</figcaption>
+        </figure>
+    </p>
+
+Feature Store script action:
+```bash
+set -e
+
+HOST="MY_INSTANCE.cloud.hopsworks.ai" # DNS of your Feature Store instance
+PROJECT="MY_PROJECT"                  # Port to reach your Hopsworks instance, defaults to 443
+HSFS_VERSION="MY_VERSION"             # The major version of HSFS needs to match the major version of Hopsworks
+API_KEY="MY_API_KEY"                  # The API key to authenticate with Hopsworks
+CONDA_ENV="MY_CONDA_ENV"              # py35 is the default for HDI 3.6
```

Comment:
I wanted to avoid that somebody leave it unchanged by accident. Fail fast was my thought. If you don't notice that you install to the wrong one then you'll be stuck debugging for a while or might have broken your default env.

Reply:
We have philosophically different users in mind - i always favour accessability over possibility of introducing hard-to-find-bug. 
Ok, your decision.